projects/cds/src/merchandising/cms-components/merchandising-carousel/merchandising-carousel.component.service.ts

```diff
@@ -75,29 +85,22 @@ export class MerchandisingCarouselComponentService {
   }
 
   sendCarouselViewEvent(
+    lastSendModelId: string,
     merchandisingCarouselModel$: Observable<MerchandisingCarouselModel>
-  ): Observable<void> {
+  ): Observable<MerchandisingCarouselModel> {
```

Comment:
Please adjust the existing test, or add a new test for this change.

Reply:
Tests was fixed.